com.unity.render-pipelines.core/Editor/Volume/VolumeEditor.cs

```diff
@@ -53,13 +79,12 @@ public override void OnInspectorGUI()
         {
             serializedObject.Update();
 
-            GUIContent label = EditorGUIUtility.TrTextContent("Mode", "A global volume is applied to the whole scene.");
             Rect lineRect = EditorGUILayout.GetControlRect();
             int isGlobal = m_IsGlobal.boolValue ? 0 : 1;
-            EditorGUI.BeginProperty(lineRect, label, m_IsGlobal);
+            EditorGUI.BeginProperty(lineRect, Styles.mode, m_IsGlobal);
             {
                 EditorGUI.BeginChangeCheck();
-                isGlobal = EditorGUI.Popup(lineRect, label, isGlobal, m_Modes);
+                isGlobal = EditorGUI.Popup(lineRect, Styles.mode, isGlobal, m_Modes);
```

Comment:
```suggestion
                isGlobal = EditorGUI.Popup(lineRect, Styles.mode, isGlobal, Styles.modes);
```

Also remove the legacy m_Modes as it have been moved in Styles

Reply:
Ahh nice catch. Missed that.

Reply:
fixed in ad7c03b